mslice/plotting/plot_window/cut_plot.py

```diff
@@ -232,25 +234,6 @@ def _has_errorbars(self):
             has_errorbars = True
         return has_errorbars
```

Comment:
This could just be `return sum(alpha) != 0`

Reply:
Alright